### PR TITLE
Leadersboard home page

### PR DESCRIPTION
**Pull request by misikirdelelew**
In this PR I have done

- plain HTML markup with minimum styling  for the leadboard
- webpack configured
- used ES6 modules

### CODE REVIEW

**Comment by MahmoudBakr23** [comment]
# Hi @misikirdelelew 😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

## **Invalid Submission**

- Please update your README according to the newest Microverse [template](https://github.com/microverseinc/readme-template)

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._ 🙏

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @misikirdelelew , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
- [ ] it would be better to make sure that the `MIT` link is working and it should take you to the License of the project in order for your project will be professional

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,207 @@
...
+  <h3><b>Leaderboard</b></h3>
+
+</div>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```
Comment (rotimiazeez) [inline README.md]: - Could you make sure the logo provided in your README.md file is visible or better still not present at all? The current logo is broken 😢

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
Hi @misikirdelelew 😉, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
- Good setup 👍🏽
- No linter error 👍🏽


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by julie-ify — APPROVED** [review]
Hi @misikirdelelew,

## Highlights!
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**